### ISSUE TITLE
Access to specific folder

### ISSUE THREAD

**Issue by Barmem** [Swati4star/Images-to-PDF#1129]
Maybe give an option to allow access to specific folder instead of whole file system?

**Comment by Tombstone2K** [comment]
> Maybe give an option to allow access to specific folder instead of whole file system?

Yes, I agree. Instead of granting access to all files on the system. Maybe android's privacy preserving file picker can be used.

Any thought ? @Swati4star 

Regards

**Comment by abhisheksharma886** [comment]
Could you please assign this issue to me? I'm interested in resolving it

**Comment by Barmem** [comment]
> Could you please assign this issue to me? I'm interested in resolving it

doesnt look like i can assign it to anyone, dont have access

**Comment by ShaneHe023** [comment]
Hi I am intersted in this issue. Can anyone assign to me?
